docs/modules/fault-tolerance/pages/backups.adoc

```diff
@@ -7,6 +7,17 @@ The distribution happens on the partition level; the data and its backups are st
 memory partitions. See the xref:overview:data-partitioning.adoc[Data Partitioning] and xref:clusters:partition-group-configuration.adoc[Partition Grouping]
 for more information about the partitioning.
 
+[IMPORTANT]
+====
+Hazelcast does not support backup redistribution on data structure level. For example, for a map with a nonzero backup count,
+
+1. increasing the backup count does **not** create additional copies of existing map entries,
```

Comment:
you do not distinguish static and dynamic config. Dynamic config cannot be changed at all

Reply:
I added a footnote to explain how the backup count can be changed. (ca51ef8c)